google-style.xml

```diff
@@ -41,9 +41,16 @@
             <property name="allowByTailComment" value="true"/>
             <property name="allowNonPrintableEscapes" value="true"/>
         </module>
-        <module name="AvoidStarImport">
-            <property name="severity" value="warning"/>
-        </module>
+        <module name="AvoidStarImport"/>
```

Comment:
is it still the same set of rules that Google uses?
does it create warnings for unused/star imports?

Reply:
> is it still the same set of rules that Google uses?

no, Google relies on the very first version of this config

> does it create warnings for unused/star imports?

it produces violations for all star(*) imports, unused and redundant imports are checked by 2 modules below:
```xml
<module name="RedundantImport"/>
<module name="UnusedImports"/>
```

Reply:
then we probably should not have `google` in its name as it is now a custom one